src/stores/cardStore.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { CardStore } from "./cardStore";
+import { Card, CardSuit } from "../types";
+
+const fakeCards: Card[] = [
+  { face: "2", suit: CardSuit.Diamond, value: 2, flipped: false },
+  { face: "9", suit: CardSuit.Clover, value: 9, flipped: false },
+  { face: "jack", suit: CardSuit.Clover, value: 0, flipped: false },
+  { face: "3", suit: CardSuit.Heart, value: 3, flipped: false },
+  { face: "4", suit: CardSuit.Heart, value: 4, flipped: false }
+];
+
+describe("create a cardStore with default settings", () => {
+  it("should return cardStore with initial settings", () => {
+    const cardStore = new CardStore();
+
+    expect(cardStore.cards).toHaveLength(288);
+    expect(cardStore.playerCards).toHaveLength(0);
+    expect(cardStore.bankerCards).toHaveLength(0);
+  });
+});
+
+describe("cardStore methods should work", () => {
+  it("should draw a card", () => {
```

Comment:
`should draw a card` but inside the test expect checks properties... Weird but ok)